test/integration/src/test/java/io/pravega/test/integration/ControllerRestApiTest.java

```diff
@@ -191,6 +196,17 @@ public void restApiTests() {
         Assert.assertEquals("List streams size", 1, response.readEntity(StreamsList.class).getStreams().size());
         log.info("List streams successful");
 
+        // Test listStream GET /v1/scopes/scope1/streams for tags
```

Comment:
It may be good to add a test for filterType=showInternalStreams

Reply:
Done